### PR TITLE
Concent integration deposit

### PR DESCRIPTION
**Pull request by jiivan**

### CODE REVIEW

File: golem/task/tasksession.py [hunk 1]
```diff
@@ -820,22 +821,15 @@ def _react_to_ack_report_computed_task(self, msg):
             ack_report_computed_task=msg,
         )
         logger.debug('[CONCENT] ForceResults: %s', delayed_forcing_msg)
-        report_computed_task = get_task_message(
-            'ReportComputedTask',
-            msg.task_id,
-            msg.subtask_id,
+        ttc_deadline = datetime.datetime.utcfromtimestamp(
```
Comment (shadeofblue) [inline golem/task/tasksession.py:824]: hmm... @jiivan wasn't the `report_computed_task` validation/logging needed here?

**Review by shadeofblue — APPROVED** [review]
looks good as far as I can tell

File: golem/task/tasksession.py (hunk 1 above)
Comment (jiivan) [reply golem/task/tasksession.py:824]: RCT was fetched from history only because it was historically absent in `AckReportComputedTask` back in the days.

Comment (shadeofblue) [reply golem/task/tasksession.py:824]: ah, gotcha

**Pull request merged**